README.md

```diff
@@ -54,11 +88,29 @@ describe('application launch', function () {
   it('shows an initial window', function () {
     return this.app.client.getWindowCount().then(function (count) {
       assert.equal(count, 1)
+      // Please note that getWindowCount() it could give you 2 windows if `dev tools` are opened.
```

Comment:
"Please note that getWindowCount() it could..." → Note that getWindowCount() will return `2` if devtools are open.